integration/spark/shared/src/main/java/io/openlineage/spark/agent/filters/DatabricksEventFilter.java

```diff
@@ -38,7 +38,7 @@ public DatabricksEventFilter(OpenLineageContext context) {
 
   @Override
   public boolean isDisabled(SparkListenerEvent event) {
-    return isSerializeFromObject() || isWriteIntoDeltaCommand() || isDisabledDatabricksPlan(event);
+    return isWriteIntoDeltaCommand() || isDisabledDatabricksPlan(event);
```

Comment:
The main problem of this class is that it's not doing what the name suggests. 
It should filter events for Databricks runtime only. 

I think `isDisabled` method should start with:
```
if (!DatabricksUtils.isRunOnDatabricksPlatform(context)  || !context.getQueryExecution().isPresent()) {
      return false;
}
```
This should solve your issue as you get an event filtered outside databricks runtime.

Reply:
The existing check is essentially doing if (!DatabricksUtils.isRunOnDatabricksPlatform(context)
        || !context.getQueryExecution().isPresent()) & contains("sql.transaction.tahoe.commands.WriteIntoDeltaCommand")) 
        
        
 Are you suggesting  we dont need to care if its WriteIntoDeltaCommand ?

Reply:
Sorry, my previous comment was inaccurate. 
I am suggesting that this class should have no effect outside databricks runtime. 

So `isDisabled` method should start with `if (!DatabricksUtils.isRunOnDatabricksPlatform(context) { return false }`

Reply:
Bcz we're close to release time, I found it's easier to include proposed change within the commit: https://github.com/OpenLineage/OpenLineage/pull/3403/commits/b31404ca2a6d55386b1bc5a211170811ed2d1428

Thanks for contributing